core-web/libs/template-builder/src/lib/components/template-builder/template-builder.component.spec.ts

```diff
@@ -116,4 +117,23 @@ describe('TemplateBuilderComponent', () => {
         spectator.component.deleteRow('123');
         expect(removeRowMock).toHaveBeenCalledWith('123');
     });
+
+    it('should call addContainer from store when triggering addContainer', (done) => {
+        const addContainerMock = jest.spyOn(store, 'addContainer');
+
+        let widgetToAddContainer: DotGridStackWidget;
+        let rowId: string;
+
+        expect.assertions(1);
+
+        store.state$.pipe(take(1)).subscribe(({ items }) => {
+            widgetToAddContainer = items[0].subGridOpts.children[0];
+            rowId = items[0].id as string;
+
+            spectator.component.addContainer(widgetToAddContainer, rowId, mockContainer);
+
+            expect(addContainerMock).toHaveBeenCalled();
+            done();
```

Comment:
I think that `expect.assertions(1)` waits for an `expect` to trigger therefore we don't need a `done` function